packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-input.component.tsx

```diff
@@ -54,10 +56,24 @@ const VitalsBiometricInput: React.FC<VitalsBiometricInputProps> = ({
   muacColorCode,
   useMuacColors,
   isWithinNormalRange = true,
+  interpretation,
 }) => {
   const { t } = useTranslation();
   const isTablet = useLayoutType() === 'tablet';
   const [invalid, setInvalid] = useState(false);
+  const [isFocused, setFocused] = useState(false);
+
+  const handleFocus = () => {
+    setFocused(true);
+  };
+
+  const handleBlur = () => {
+    setFocused(false);
+  };
+
+  const flaggedCritical =
+    interpretation && ['critically_low', 'critically_high', 'high', 'low'].includes(interpretation);
+  const flaggedAbnormal = interpretation && interpretation !== 'normal';
```

Comment:
Aren't these two conditions basically equivalent?

Reply:
I wouldn't expect to see `high` and `low` included in the array when checking for vitals that are flagged as critical.

Reply:
@ibacher @denniskigen According to the zero height design, the one arrow is for `low` or `high` and the double arrows are for `critical values`, maybe the naming is the wrong one but it is doing the right thing.